example/spec/parallel/ingest/ingestFromPdrWithExecutionNamePrefixSpec.js

```diff
@@ -167,7 +176,7 @@ describe('The DiscoverAndQueuePdrsExecutionPrefix workflow', () => {
   it('properly sets the name of the queued execution', () => {
     if (beforeAllFailed) fail('beforeAll() failed');
     else {
-      const executionArn = queuePdrsOutput.payload.running[0];
+      executionArn = queuePdrsOutput.payload.running[0];
```

Comment:
Should we be setting the 'global' test variable here and on L190?     `executionArn` looks like a value that should be set in the beforeAll where queuePdrsOutput is obtained.

Reply:
Yep! [`89cd3a5` (#2254)](https://github.com/nasa/cumulus/pull/2254/commits/89cd3a567602e004c381fa52223a15ec3540b02b)